Header header

uint64 sequence
uint64 analog_1
uint64 analog_2
uint64 analog_3
uint64 analog_4
uint64 analog_5
uint64 analog_6
uint64 analog_7
uint64 analog_8
bool digital_1
bool digital_2
bool digital_3
bool digital_4

